docs/guides/designing-the-delta-format.md

```diff
@@ -123,7 +123,7 @@ var content = [
 
 But if the answer is yes, then we violate the canonical constraint since any permutation of characters having an align attribute would represent the same content.
 
-So we cannot just naively get rid of the newline character. We have to also either get rid of line attributes, or expand line attributes to fill all characters on the line. But what if we deleted the newline from this:
+So we cannot just naively get rid of the newline character. We also have to either get rid of line attributes, or expand them to fill all characters on the line. But what if we removed the _newline_ from it:
```

Comment:
Sure. The reason why I've used _italics_ was to change the tone, rather than to apply emphasis. But for the sake of consistency, I'll do as you ask.